users/views.py

```diff
@@ -35,6 +36,34 @@ def activate(request, uidb64, token):
             {'message': 'Your link is invalid!'}))
 
 
+def users_detail(request, user_id):
+    user = get_user_model().objects.get(pk=user_id)
+    change_permission = False
+    if request.user.email == user.email or request.user.is_staff:
+        change_permission = True
+    return render(request, 'user_page.html', {
+        'user': user,
+        'address': user.get_address(),
+        'name': user.get_full_name(),
+        'change_permission': change_permission})
+
+
+class UpdateProfileView(generics.UpdateAPIView):
+    permission_classes = (permissions.IsAuthenticated,)
+    serializer_class = ChangeProfileSerializer
+
+    def get_queryset(self):
+        user = get_user_model()
```

Comment:
'user' is a class, not instance, so it should be upper case:
User = get_user_model()